apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -207,22 +232,13 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
         return;
       }
 
-      if (newName.length > 0) {
-        dispatch(
-          ConversationsActions.updateConversation({
-            id: conversation.id,
-            values: {
-              name: newName,
-              isNameChanged: true,
-            },
-          }),
-        );
-
+      if (newName !== conversation.name) {
```

Comment:
We should chek if it was shared/published and show confirmation only in this case